content/december-2014-meet-minutes.md

```diff
@@ -55,11 +55,11 @@ see people forming groups and discussing various topics.
 ## Lightning Talks - II
 
 After the networking tea break, we had a second round of lightning
-talks. 
+talks.
 
 <a
-href="http://photos1.meetupstatic.com/photos/event/7/0/1/b/highres_432628699.jpeg"><img
-src="http://photos1.meetupstatic.com/photos/event/7/0/1/b/event_432628699.jpeg"
+href="http://photos2.meetupstatic.com/photos/event/7/0/3/a/event_432628730.jpeg"><img
+src="http://photos2.meetupstatic.com/photos/event/7/0/3/a/event_432628730.jpeg"
```

Comment:
OK, I now understand what you are referring to. I am getting the same issue as well. But is this a transient issue? If yes then we might want to rethink our strategy of linking to external resources. Instead we could pull in the external resources once using bower, and serve them from a local copy.